finetune_adapter.py

```diff
@@ -78,10 +74,10 @@ def main(
 
     optimizer = torch.optim.AdamW(model.parameters(), lr=learning_rate, weight_decay=weight_decay)
     model, optimizer = fabric.setup(model, optimizer)
-    train(fabric, model, optimizer, train_data, val_data, checkpoint_dir)
+    train(fabric, model, optimizer, train_data, val_data, checkpoint_dir, out_dir)
 
     # Save the final checkpoint at the end of training
-    save_model_checkpoint(fabric, model, checkpoint_dir / "lit_model_adapter_finetuned.pth")
```

Comment:
nit: all other files use underscore instead of dashes (`lit_model` , `lit_config`, `tokenizer_config`). The adapter checkpoint should too

Reply:
That's not true, I was always consistent and we have all checkpoints in lit-llama like this. For example see finetune_adapter.py in lit-llama, the same naming there. This does not matter to the practitioner and I'm not going to manually rename all the files we already have just to make this consistent.